pkg/atomic2/atomic_test.go

```diff
@@ -40,7 +40,6 @@ func (t *testAtomicSuite) TestAtomicError(c *check.C) {
 	err2 = e.Get()
 	c.Assert(err2, check.DeepEquals, err)
 
-	err = errors.New("test2")
```

Comment:
I copied from chunzhu's code, guess he want to check `e.Set(err)` doesn't get a reference to `err`

Reply:
I added it back and a check with it.